img_styler/image_prompt/control_net/annotator/hed/__init__.py

```diff
@@ -0,0 +1,132 @@
+import numpy as np
+import cv2
+import os
+import torch
+from einops import rearrange
+from annotator.util import annotator_ckpts_path
+
```

Comment:
Question: Were proper references to other repos/original paper on ControlNet included?

Reply:
Not yet. I might be able to reduce the number of files needed from the ControlNet repo so I thought I'd add the references afterward.